src/injectscripts/jobs/view.js

```diff
@@ -398,12 +335,18 @@ function InstantTaskButton() {
                 if (vv.TeamLeader)
                 {
                   item = return_li(v.Id,v.Callsign.toUpperCase(),vv.Person.FirstName+" "+vv.Person.LastName,v.TaskedJobCount+"");
-                  $(item).click(function () {
-                    TaskTeam(v.Id)
-                  })
                 }
               })
             }
+            //still create teams that have no TL
+            if (typeof(item) == "undefined") {
```

Comment:
consider setting `item` to `null` or something on L328 and then check if it's null rather than using this ugliness.